### PR TITLE
Add salt.pm into console_tests for SLES 15

### PR DESCRIPTION
**Pull request by Zaoliang**
- add salt.pm for SLES 15 in main.pm
- see poo#26812
- verification run:
	http://e13.suse.de/tests/61#step/salt

### CODE REVIEW

File: products/sle/main.pm [hunk 1]
```diff
@@ -804,8 +804,11 @@ sub load_consoletests {
     loadtest "console/curl_https";
     if (check_var_array('SCC_ADDONS', 'asmm')) {
         loadtest "console/puppet";
+    }
+    if (check_var_array('SCC_ADDONS', 'asmm') || sle_version_at_least('15')) {
         loadtest "console/salt";
     }
+
```
Comment (okurz) [inline products/sle/main.pm]: please delete blank line. We don't have this as a separator in the other places around so we should not have it here

**Comment by okurz** [comment]
Your verification run failed in salt: http://e13.suse.de/tests/61#step/salt/7 so why did you create this PR then?

I checked for the package "salt-master" and found it in https://build.suse.de/package/view_file/SUSE:SLE-15:GA/000product/unsorted.yml?expand=1 so it is currently not in module. Please create a bug about this. This is why we do all this testing :)

@rwx788

File: products/sle/main.pm (hunk 1 above)
Comment (Zaoliang) [reply products/sle/main.pm]: blank line got removed now

**Comment by okurz** [comment]
*bump*

**Comment by Zaoliang** [comment]
we have already a bug for salt, it has to do with ruby upgrade. will add the bugID later...
@okurz @rwx788 this ticket is about to add salt test for sles 15. not about to fix bug.

**Comment by Zaoliang** [comment]
bug has been already filed:
https://bugzilla.opensuse.org/show_bug.cgi?id=1069711

**Comment by okurz** [comment]
so I was right that we should not have merged this in before: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/4011

**Comment by okurz** [comment]
The bug you mentioned is not valid for SLE15 but only valid for openSUSE Tumbleweed at the moment. For SLE15 the package salt-master is not part of any default module.

**Comment by Zaoliang** [comment]
@okurz filed new one:

https://bugzilla.suse.com/show_bug.cgi?id=1071451

**Pull request merged**